spec/core/integration/EnvSpec.js

```diff
@@ -2138,6 +2138,25 @@ describe("Env integration", function() {
       });
     });
 
+    describe('When failSpecWithNoExpectations is enabled and spec has no expectations', function() {
```

Comment:
Could you please add a similar integration test for the case where failSpecWithNoExpectations is not enabled?